pytradfri/api/aiocoap_api.py

```diff
@@ -130,6 +152,8 @@ def error_callback(ex):
         ob = pr.observation
         ob.register_callback(success_callback)
         ob.register_errback(error_callback)
+        nonlocal _observations_err_callbacks
```

Comment:
I don't think that you have to put a `nonlocal` to access variables in an outer clojure.